src/docs/flex-cli/manage-search-schemas-with-flex-cli/index.md

```diff
@@ -29,11 +29,18 @@ make those filters work as expected.
 
 ## Extended data types and schema scopes
 
-There are various kinds of extended data. Listings support public data,
-private data, and public metadata. All these are editable in Console by
-the operator, but only public data and metadata can be seen by other
-users. For this reason, search schemas can be scoped either to `public`
-or `metadata`. To see more details about extended data, see the
+There are various kinds of extended data. Search schema support differs depending on whether you
+use Marketplace API or Integration API, and on whether the `schema-for` value is `listing` or `user`.
+The following table summarizes the supported search schema scopes.
+
+|          | Marketplace API                | Integration API                  |
+| -------- | ------------------------------ | -------------------------------- |
+| listing  | public, meta, keyword          | public, meta, keyword            |
+| user     | N/A                            | public, meta, private, protected |
+
+Extended data are editable in Console by the operator, but only public data and 
+metadata can be seen by other users. For this reason, search schemas can be scoped 
+either to `public` or `metadata`. To see more details about extended data, see the
```

Comment:
The table implies that `user` search schemas can be scoped to `private` and `protected` as well, so this sentence should be reformulated somehow.

Reply:
Yes, true. Didn't change here much as I didn't use console for extended data. I will verify the situation.

Reply:
The way I read this, the paragraph talks about search schemas in general, not separating them between `listing` and `user`. I'd suggest making the distinction explicit - something along the following lines:

"All types of extended data are editable in Console by the operator, but only public data and 
metadata can be seen by other marketplace users. For this reason, listing search schemas can be scoped  either to `public` or `metadata`. User search schemas are currently only available through an Integration API endpoint, so they can also be scoped to `private` and `protected`."

Reply:
@bladealslayer 
I more or less adopted @SariSaar 's suggestion but I took out `For this reason` because there is no strong causal relationship. `listing's `private`, `protected` data schema should be enabled in integration API as well`. And I used the word `Currently` to imply situation could change.